crates/e2e/tests/e2e/onchain_settlement.rs

```diff
@@ -82,17 +82,25 @@ async fn onchain_settlement(web3: Web3) {
 
     let services = Services::new(onchain.contracts()).await;
     let solver_endpoint = colocation::start_naive_solver().await;
+    let quoter_endpoint = colocation::start_baseline_solver(solver.address()).await;
     colocation::start_driver(
         onchain.contracts(),
-        vec![SolverEngine {
-            name: "test_solver".into(),
-            account: solver,
-            endpoint: solver_endpoint,
-        }],
+        vec![
```

Comment:
Why is this change needed?

Reply:
We don't verify native price quotes because for those we can't expect to actually have the needed balance in the settlement contract. Also we didn't implement gas estimates for the naive solver since it's not intended to be used as a price estimator. Now that we discard all quotes that can't be verified and don't have a gas price estimate the naive solver is no longer able to compute the native prices for the sell and buy token that are needed to create an order.